source/cppexpose/include/cppexpose/reflection/AbstractProperty.h

```diff
@@ -87,6 +87,15 @@ class CPPEXPOSE_API AbstractProperty : public AbstractTyped
 
     /**
     *  @brief
+    *    Finds out whether the Object has a parent
+    *
+    *  @return
+    *    false if the parent is null, otherwise true
```

Comment:
No. Until now I used "null" instead of "NULL" or "nullptr" in the entire documentation. We can decide to replace it everywhere, but we should not begin to mix it. So please stay with "null" for now, you used it consistently with the rest of the documentation.